services/scoop/scoop.tester.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
```

Comment:
Same comment as above regarding the filename (`scoop-version.tester.js`)

Reply:
Also done in 97897c3f5001443a09e3677c8b660bbdd3daeab6.